redash/query_runner/mongodb.py

```diff
@@ -130,11 +130,31 @@ class MongoDB(BaseQueryRunner):
     @classmethod
     def configuration_schema(cls):
         return {
-            "type": "object",
-            "properties": {
-                "connectionString": {"type": "string", "title": "Connection String"},
-                "dbName": {"type": "string", "title": "Database Name"},
-                "replicaSetName": {"type": "string", "title": "Replica Set Name"},
+            'type': 'object',
+            'properties': {
+                'connectionString': {
+                    'type': 'string',
+                    'title': 'Connection String'
+                },
+                'dbName': {
+                    'type': 'string',
+                    'title': "Database Name"
+                },
+                'replicaSetName': {
+                    'type': 'string',
+                    'title': 'Replica Set Name'
+                },
```

Comment:
@arikfr I'm wondering why restyled didn't catch this change of style if black would have turned the single quotes into double quotes?

Reply:
🤷 no idea.